build/gen_stub.php

```diff
@@ -83,9 +83,15 @@ function processStubFile(string $stubFile, Context $context, bool $includeOnly =
             }
         }
 
+        /* As exit() and die() are proper token/keywords an need to hack-around */
```

Comment:
Not sure if this is a grammatically correct sentence.

Reply:
I'll fix it in the commit where I add the NEWS/UPGRADING entry